examples/demo/Standard_Editors/TextEditor_demo.py

```diff
@@ -55,19 +53,21 @@ class TextEditorDemo(HasTraits):
     text_pass_group = Group(
         Item('password', style='simple', label='Simple'),
         # custom and text style are the same as simple, not shown.
+        # readonly style is not useful for passwords.
         label='Password'
     )
 
     # The view includes one group per data type. These will be displayed
     # on separate tabbed panels:
-    traits_view = View(
+    view = View(
```

Comment:
Hmmmm I think defining a `default_traits_view` method is the recommended style nowadays. I might be mistaken though. @corranwebster Could you confirm please?

Reply:
From offline discussion - static `view` definition is preferred for demos

Reply:
Offline discussion: Not true. Both static and dynamic ways are good.
I am not sure whether it should be `traits_view` or `view` though.

Reply:
Yes, sorry. Both are good, but for simple cases static `view` definition is preferred.

As for `traits_view`, I think it's best to rename it because of https://github.com/enthought/traits/issues/705

Reply:
Apparently one can call it anything they like and the last one standing is used. Here is the relevant magic:
https://github.com/enthought/traits/blob/e73cae932cca147661a8f2aedb0489044cbe9e2e/traits/has_traits.py#L498-L500

Reply:
I believe `traits_view` will need to be whitelisted for https://github.com/enthought/traits/issues/705 because it is explicitly advertised by traits:
https://github.com/enthought/traits/blob/e73cae932cca147661a8f2aedb0489044cbe9e2e/traits/has_traits.py#L105-L106
https://github.com/enthought/traits/blob/e73cae932cca147661a8f2aedb0489044cbe9e2e/traits/has_traits.py#L1704-L1709

So we have two views defined on this class, one is called `traits_view` and the other one called `view`. The one assigned to `traits_view` will be used. 
Just to add to that fun, if `default_traits_view` is also defined, then `default_traits_view` trumps the rest.

This is getting a bit orthogonal. Since the original example use `traits_view` and the `view` name is never advertised by the API (looks like), shall we use `traits_view` here?

Reply:
Okay, I'll use `traits_view` in all demos and if a time comes for that to change, renaming should be easy via find-replace